.github/workflows/compatibility.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       fail-fast:    false
       max-parallel: 10
       matrix:
-        woocommerce: [ '4.4.0', '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0' ]
+        woocommerce: [ '4.4.0', '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0' ]
```

Comment:
That list looks scary, but I don't have good suggestions.

Reply:
My suggestion is we start bringing that inline with our L-2 support policy starting in the New Year.

Reply:
Sounds like a good plan.